README.md

```diff
@@ -76,4 +76,80 @@ $ singularity build osgvo-pilot.sif docker://opensciencegrid/osgvo-docker-pilot
 ```
 
 
+## CVMFS Without a Bind-Mount Using cvmfsexec
+
+If you don't have CVMFS available on the host, the container can still make
+CVMFS available by using [cvmfsexec](https://github.com/cvmfs/cvmfsexec#readme).
+
+This will require a kernel version >= 3.10.0-1127 on an EL7-compatible host
+or >= 4.18 on an EL8-compatible host, or user namespaces enabled by hand --
```

Comment:
```suggestion
This will require a kernel version >= 3.10.0-1127 on an EL7-compatible host with user namespaces enabled
or >= 4.18 on an EL8-compatible host --
```